modules/formulize/include/functions.php

```diff
@@ -6817,15 +6817,12 @@ class='toggle-button-change-columns'
 						".$formObject->getVar('title')." <span id='arrow-up' aria-label='up arrow'>&and;</span><span id='arrow-down' aria-label='down arrow'>&or;</span>
 					</button>
 				";
-				$html .= "
-					<div
-						class='elements-checkbox-list'
-						id='id-change-columns-toggle-panel-".$thisFid."'
-						aria-labelledby='id-change-columns-toggle-header-".$thisFid."'
-					>
-						".$boxeshtml."
-					</div>
-				";
+				$html .="<fieldset class='elements-checkbox-fieldset'>";
```

Comment:
No 'aria-labelledby' in the new version of this, I assume that's on purpose? Just checking.

Reply:
Ahh! Good catch. The aria tag should probably be there.

Reply:
Oh wait... I need to double check on this.

Reply:
Ahh I see. This was the backport of accessible toggle work I believe.